.github/workflows/README.md

```
@@ -0,0 +1,98 @@
+#### 3. Copilot (Issue Auto-Assign)
+```bash
+# Option A: Invite 'Copilot' user as collaborator
+# Settings → Collaborators → Add people → Search: Copilot
+
+# Option B: Edit ai-copilot-assign.yml line 35 to use different assignee
+
+# Done! New issues auto-assigned to Copilot
+```
+
+### Full Documentation
+See [`docs/AI_WORKFLOWS.md`](../../docs/AI_WORKFLOWS.md) for:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)